getting-started.rst

```diff
@@ -46,3 +53,28 @@ This requires:
   - Forms can be uploaded directly to Collect, without using Aggregate
 
   ODK is a very flexible set of tools, and orgnizations will find their own best practices for adopting it.
+
+.. _installing-collect:
+
+Installing Collect
+---------------------
+
+- **Recommended:**`The ODK Collect App is available in the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
+- You can also download from the web and install manually:
+
+  - From your device's application drawer, choose :guilabel:`Settings`, then :guilabel:`Applications`. Make sure Unknown sources is checked.
+  - Return to the application drawer and choose :guilabel:`Browser`. Navigate to `https://opendatakit.org/downloads/download-category/collect/ <https://opendatakit.org/downloads/download-category/collect/>`_ and download the ODK Collect APK.
+  - In the download window, you will see ODK_Collect_vN.N.N.apk. - Select it to download the file.
+
+    - On older devices, the APK will automatically install after you approve the security settings.
+    - On newer devices, you must go to the download list, rename the file to restore the .apk extension (the extension will have been renamed to .man during the download process), then click on it to install it.
+
+.. warning::
+
+  ODK Collect requires a external SD Card. Most current Android models have these built in, but some models (mostly Android 2.x and earlier) do not and will need one to be installed.
```

Comment:
> ODK Collect requires a external SD Card

 please see this discussion: https://forum.opendatakit.org/t/sd-card-requirement/6128/6

> Most current Android models  

I think devices is a better word compared to models